pkgs/top-level/all-packages.nix

```diff
@@ -36342,4 +36342,6 @@ with pkgs;
   swift-corelibs-libdispatch = callPackage ../development/libraries/swift-corelibs-libdispatch { };
 
   swaysettings = callPackage ../applications/misc/swaysettings { };
+
+  lighthouse = callPackage ../applications/blockchains/lighthouse { };
```

Comment:
Please (roughly) alphabetically sort this, don't toss it at the bottom of the file.